tests/std/tests/P0896R4_ranges_alg_unique_copy/test.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <span>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+using P = pair<int, int>;
+
+// Validate that unique_copy aliases in_out_result
+STATIC_ASSERT(same_as<ranges::unique_copy_result<int, double>, ranges::in_out_result<int, double>>);
+
+// Validate dangling story
+STATIC_ASSERT(same_as<decltype(ranges::unique_copy(borrowed<false>{}, static_cast<int*>(nullptr))),
+    ranges::unique_copy_result<ranges::dangling, int*>>);
+STATIC_ASSERT(same_as<decltype(ranges::unique_copy(borrowed<true>{}, static_cast<int*>(nullptr))),

```

Comment:
Is this a chance to use the new `nullptr_to<int>` helper?

Reply:
Thanks, lazyme didnt check whether it was already merged